docs/tutorials/tutorials/export-pytorch-model.md

```diff
@@ -1,185 +1,325 @@
+def my_trace(g, self):
+    return g.op("ai.onnx.contrib::Trace", self)
 
-# Run the exported model with ONNX Runtime
-ort_sess = onnxruntime.InferenceSession(f.getvalue())
-ort_inputs = dict((ort_sess.get_inputs()[i].name, input.cpu().numpy()) for i, input in enumerate((x,)))
-ort_outputs = ort_sess.run(None, ort_inputs)
+torch.onnx.register_custom_op_symbolic('::trace', my_trace, 1)
 
-# Validate PyTorch and ONNX Runtime results
-numpy.testing.assert_allclose(pt_outputs.cpu().numpy(), ort_outputs[0], rtol=1e-03, atol=1e-05)
+# Export the model
+input = torch.randn(3, 3)
+torch.onnx.export(model, (input, ), "model_with_trace.onnx", verbose=True)
 ```
 
-By default, the opset version will be set to ``1`` for custom opsets. If you'd like to export your
-custom op to a higher opset version, you can specify the custom opset domain and version using 
-the ``custom_opsets argument`` when calling the export API. Note that this is different than the opset 
-version associated with default ```ONNX``` domain.
+## Run a model with custom operators using the Python inference API
+
+Again, once the model has been created and exported, you can use the ONNX Runtime inference
+APIs to run the model. Two steps are required before using the ONNX Runtime inference APIs.
+Note that these two steps must be executed in this order.
+
+1. Provide a definition for the operator.
+   This is achieved by the `@onnx_op` annotation, which adds your operator to the
+   library of custom operators already in the extensions library.
+2. Register the operator definition with ONNX Runtime
+   The previous step added the operator to the extensions library. This step
+   makes all operators in the extensions library, as well as your new
+   operator available to ONNX Runtime.
 
 ```python
-torch.onnx.export(CustomInverse(), (x,), f, custom_opsets={"com.microsoft": 5})
+import numpy
+import torch
+import onnxruntime
+import onnxruntime_extensions
+from onnxruntime_extensions import PyOp, onnx_op
+
+# Define the operator
+@onnx_op(op_type="Trace", inputs=[PyOp.dt_float])
+def trace(x):
+    return numpy.trace(x)
+
+# Register the extensions library (of which your operator is now effectively a member)
+so = onnxruntime.SessionOptions()
+so.register_custom_ops_library(onnxruntime_extensions.get_library_path())
+
+sess = onnxruntime.InferenceSession("model_with_trace.onnx", so)
+input_name = sess.get_inputs()[0].name
+output = sess.run(None, {input_name: torch.randn(3, 3).numpy()})[0]
+print(output)
 ```
 
-Note that you can export a custom op to any version >= the opset version used at registration.
+## Run a model with extension operators using the C++ API
 
-We have a set of tests for export and output validation of ONNX models with ONNX Runtime custom ops in 
-``tools/test/test_test_custom_ops_pytorch_exporter.py``. If you're adding a new custom operator, please
-make sure to include tests in this file.
+1. Install the ONNX Runtime C++ libraries
```

Comment:
Should we mention that this command lines is for Linux but we support Windows and MacOS platforms as well?